scripts/database/create_tables.sql

```diff
@@ -25,7 +25,7 @@ BEGIN
       EARLY_RECALL_DATE                 DATE                 null,
       CALL_RECALL_STATUS_AUTHORISED_BY  VARCHAR(200)         null,
       END_CODE_ID                       BIGINT               null,
-      END_CODE_LAST_UPDATED             DATETIME             null,
+      END_CODE_LAST_UPDATED             DATE                 null,
```

Comment:
I spoke to Vivek about END_CODE_LAST_UPDATED, it should be a DATETIME in both the EPISODE and the PARTICIPANT_SCREENING_EPISODE tables